changes/pr0000.yaml

```
@@ -0,0 +1,5 @@
+fix:
+  - "Added correct Databricks API params (autoscale, policy_id) - [Databricks API](https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsCreate)"
```

These file will be used to populate the release notes for the next release. We like to include links to the PR that made the change so I've made a suggestion to include the link to this PR and tweak the working. Also, the name of this file should be `pr5681.yaml`.
```suggestion
  - "Corrected Databricks new cluster API params: autoscale and policy_id - [#5681](https://github.com/PrefectHQ/prefect/pull/5681)"
```